src/app/views/app_frame/frame/main/side_panel/note_bar/notes/helpers/notes-api.js

```diff
@@ -68,12 +70,23 @@ const deleteNote = async ({ id, accessToken, setNotes, setApiLoading }) => {
 }
 
 const editNote = async ({ id, newContent, accessToken, setNotes, setApiLoading }) => {
-  const response = await axios.put( `https://internmatch-cyrus.gada.io/v7/notes/${id}`, {
-    content: newContent,
-  })
+  setApiLoading( true )
+
+  const response = await axios.put( `https://internmatch-cyrus.gada.io/v7/notes/${id}`,
+    {
+      content: newContent
+    },
+    {
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+      },
+    },
+  )
 
   getAll({ setNotes, accessToken, setApiLoading })
```

Comment:
we should consider not getting all the notes again, the response should hold all the information we need to update the relevant note (or not)